troubleshooting.md

```diff
@@ -240,7 +247,7 @@ cannot find newuidmap: exec: "newuidmap": executable file not found in $PATH
 
 #### Solution
 
-Install a version of shadow-utils that includes these executables.  Note RHEL7 and Centos 7 will not have support for this until RHEL7.7 is released.
+Install a version of shadow-utils that includes these executables.  Note RHEL7 and CentOS 7 will not have support for this until RHEL7.7 is released.
```

Comment:
extra space ` Note RHEL7`

Reply:
there are `RHEL7` and `RHEL 7`, can we use the same format, with space before 7 or without space

Reply:
Yup